CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
   - `bytes_received` is now `network_received_bytes`.
   - `bytes_sent` is now `network_sent_bytes`.
 - Remove `last_throughput_measurement_timestamp`, `avg_bps_in` and `avg_bps_out` metrics exposed by the Prometheus exporter.
+- Prometheus metrics `network_sent_bytes` and `network_received_bytes` update on every message instead of during "housekeeping".
```

Comment:
I don't think this is particularly useful for a user of the node. Maybe state something along the lines of what you put in the PR description.

Reply:
yes, I agree, it should provide more information.